<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iAutoRigWidget</class>
 <widget class="QWidget" name="iAutoRigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iAutoRig</string>
  </property>
  <widget class="QLineEdit" name="BodyPartName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="HumanoidButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Humanoid</string>
   </property>
  </widget>
  <widget class="QLabel" name="BodyPartNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>195</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Enter Body Part Name For Desired Parameter Name : </string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ibra's Rigging Toolkit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HumanoidInstructions">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute Humanoid Auto Rig On Named Bones : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateArmButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Arm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SubnetPath">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="PathTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Drag and drop rig subnet to get path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateLegButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Leg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateFingerButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Simple FK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateSpineButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Spine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateHeadAndNeckButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Head and Neck</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateHipButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Hip</string>
   </property>
  </widget>
  <widget class="QLabel" name="UtilitiesLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilities : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="MirrorLimbButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror Bone</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MirrorFingersButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror Root</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreatePistonButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Piston</string>
   </property>
  </widget>
  <widget class="QLabel" name="QuickConstraintsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Quick Constraints :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateHandsFingersButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>280</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Hand's Fingers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateQuadLegButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Quad Leg</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
